OPHD/UI/StringTable.cpp

```diff
@@ -115,6 +120,16 @@ void StringTable::computeRelativeCellPositions()
 
 		columnOffset += columnWidths[column] + mHorizontalPadding;
 	}
+
+	if (mCells.size() == 0)
+	{
+		mScreenRect.size({ 0, 0 });
+	}
+	else 
+	{
+		mScreenRect.width = static_cast<int>(mCells.back().textRelativePosition.x + columnWidths.back());
+		mScreenRect.height = static_cast<int>(mCells.back().textRelativePosition.y + rowHeights.back());
```

Comment:
This could potentially be written as:
```cpp
const auto lastCellSize = NAS2D::Vector{columnWidths.back(), rowHeights.back()}.to<int>();
mScreenRect.size(mCells.back().textRelativePosition.to<int>() + lastCellSize);
```

If cell position and sizes were converted to `int`, the above could have the type conversions removed. Recently most controls were converted to use `int` based coordinates, so that may make sense to do. That is particularly true for controls containing text, as off pixel values may cause the text to appear fuzzy.

Additionally, the name `textRelativePosition` could be simplified to `textOffset`. Normally a "position" is a `Point`, while an "offset" is a `Vector`. The data type here is `Vector`.

Reply:
Name update looks good. I like the proposed shorter name of 'textOffset'. 

Not my fault someone switched all the code to int instead of float for positions after StringTable's creation. :)

I'll look at using int for `textOffset`. If it gets too big, we can move to a separate branch, but think it will be easier.